### PR TITLE
FIT-503. Apply linting from shared library.

### PR DESCRIPTION
**Pull request by LotharII**
#### Which User story does this relate to (link)?
https://osi-cwds.atlassian.net/browse/FIT-503

#### Brief feature description
Apply linting from the shared library.

### CODE REVIEW

**Comment by LotharII** [comment]
@kelfink This PR should work fine. The problem in a previous one was in missing docker image for the push to dockerhub. I requested @mariamcws for review.

File: Jenkinsfile [hunk 1]
```diff
@@ -53,14 +53,12 @@ node(node_to_run_on()) {
...
-        app = docker.build("${DOCKER_GROUP}/${DOCKER_IMAGE}:${newTag}", "-f docker/web/Dockerfile .")
+         app = buildDockerImageForTest('./docker/web/Dockerfile')
       }
 
       app.withRun("--env CI=true") { container ->
         stage('Lint') {
-          sh "docker exec -t ${container.id} yarn lint"
-          sh "docker exec -t ${container.id} bundler-audit"
-          sh "docker exec -t ${container.id} brakeman"
+          lint()
```
Comment (mariamcws) [inline Jenkinsfile]: lint() will cover the sh "docker exec -t ${container.id} yarn lint"  
Do we still need to run these tests 
sh "docker exec -t ${container.id} bundler-audit"
sh "docker exec -t ${container.id} brakeman"

Comment (kelfink) [reply Jenkinsfile]: Agreed, unless the shared library is planning to execute all three for us.

Comment (LotharII) [reply Jenkinsfile]: fixed

**Comment by LotharII** [comment]
@kelfink Hi, please review the fixed version.

**Comment by kelfink** [comment]
Looks good.  Next time, the commit messages should be distinct, so we can see the reasons... instead of  just "FIT-503. Apply linting from shared library." every time

**Pull request merged**